util/util.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"log"
+	"net/url"
+	"path"
+)
+
+// JoinURL ...
+func JoinURL(baseURL string, pathPieces ...string) string {
+	url, err := url.Parse(baseURL)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	url.Path = path.Join(url.Path, path.Join(pathPieces...))
```

Comment:
What's the matter with `path.Join(baseURL, pathPieces...)`, why do we need a custom util for this?